### PR TITLE
fix: use dynamic arch for opentofu binary downloads

### PR DESCRIPTION
**Pull request by pat-s**

### CODE REVIEW

File: Containerfile.multiarch [hunk 1]
```diff
@@ -26,7 +26,8 @@ ENV TOFU_VERSION="${TOFU_VERSION:-v1.8.6}"
 
 RUN apk --update add --virtual .build-deps libarchive-tools && \
     apk add --no-cache curl git openssh-client && \
-    curl -SsfL "https://github.com/opentofu/opentofu/releases/download/${TOFU_VERSION}/tofu_${TOFU_VERSION##v}_linux_amd64.zip" | \
+    ARCH=$(apk --print-arch | sed 's/x86_64/amd64/; s/aarch64/arm64/') && \
```
Comment (xoxys) [inline Containerfile.multiarch:29]: Good catch. Please utilize `TARGETARCH` see https://gitea.rknet.org/container/alpine-tools/src/branch/main/Containerfile.multiarch#L39

**Review by xoxys — CHANGES_REQUESTED** [review]

**Review by xoxys — APPROVED** [review]

**Comment by xoxys** [comment]
Thanks!

**Comment by pat-s** [comment]
A new release would be appreciated 💙️

**Comment by xoxys** [comment]
Sure. Release triggered.

**Pull request merged**